docs/migrating/pgbouncer.md

```diff
@@ -40,7 +40,7 @@ select
   usename,
   application_name
 from pg_stat_activity
-where application_name ilike '%Supavisor%'
+where application_name = 'Supavisor'
```

Comment:
there's no point in doing `ilike` when we set it as `Supavisor`.

Reply:
There is also a `Supavisor auth_query` one